src/components/console/RequestList/RequestList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Item from './Item'
+import { useConsoleContext } from '../../../context/ConsoleContext'
+import { useStyles } from './RequestList.style'
+import cross from '../../../icons/cross.svg'
+
+function Test() {
+
+    const styles = useStyles()
+    const {consoleData,deleteHistory} = useConsoleContext()
+    const {requests} = consoleData
+
+    return (
+        <div className={styles.root} >
+                <div id='scroll' className={styles.content}>
+                    
+                    {requests.map(r => {
+                        return <Item {...r} key={Math.random()}/>
+                    })}
+                    
+                </div>
```

Comment:
Space again. Do you need it here?

Reply:
Also what `r` means? Please properly name the variables so anyone could simply understand during reading your code, what is going on there.

Reply:
this all comments is done

Reply:
LGTM